libs/application/templates/children-residence-change/src/fields/Confirmation/index.tsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react'
+import { useIntl } from 'react-intl'
 import { Box } from '@island.is/island-ui/core'
-import { confirmation } from '../../lib/messages'
-import { DescriptionText } from '../components'
+import { confirmation, copyUrl } from '../../lib/messages'
+import { DescriptionText, CopyUrl } from '../components'
 
 const Confirmation = () => {
+  const { formatMessage } = useIntl()
   return (
-    <Box marginTop={3}>
-      <DescriptionText
-        text={confirmation.general.description}
-        format={{ applicationNumber: '12345' }}
-      />
-    </Box>
+    <>
+      <Box marginTop={3}>
+        <DescriptionText
+          text={confirmation.general.description}
+          format={{ applicationNumber: '12345' }}
```

Comment:
Not in scope for this PR, but will we have this for parent a? Or will this only be availble when both parents have submitted?

Reply:
No we probably won't have this for parent A since we don't submit the application to the 'Starfskerfi' until parent B has signed.